src/clue/components/clue-app-header.sass

```diff
@@ -70,9 +70,14 @@ $member-size: 18px
     align-items: center
     justify-content: flex-end
 
-    .version
-      padding-right: $padding
-      font-size: 0.9em
+    .network-status-and-version
+      display: flex
+      flex-direction: column
+      justify-content: space-between
+      gap: 5px
+
+      .version
+        font-size: 12px
```

Comment:
Not required, but consider when updating SASS files converting them to SCSS; we're eventually wanting to get all the SASS updated.